x-pack/packages/kbn-langchain/server/language_models/simple_chat_model.ts

```diff
@@ -98,20 +98,13 @@ export class ActionsClientSimpleChatModel extends SimpleChatModel {
     if (!messages.length) {
       throw new Error('No messages provided.');
     }
-    const formattedMessages = [];
-    if (messages.length >= 2) {
```

Comment:
This was silly. The else just repeats the same behavior that would otherwise happen within the `forEach` loop on an entry with `messages.length === 1`.